src/components/questionary/QuestionaryDetails.tsx

```diff
@@ -30,13 +30,15 @@ export interface TableRowData {
 }
 function QuestionaryDetails(
   props: {
-    questionaryId: number;
+    questionaryId: number | undefined;
```

Comment:
I think that `undefined` is not a valid questionaryId value here. Do we need to have it like this?

Reply:
Let me double check this I think I did it mainly because typescript was complaining about some modifications in the backend after I changed some resolvers to be nullable: true

Reply:
I made the questionary nullable in the resolvers because it was needed for example if you do not have access to get questionary on a proposal but just proposal data it needs to be nullable.